File: src/task_1/runner.cpp
#include <geometry_msgs/PointStamped.h>
#include <ros/ros.h>

int main(int argc, char **argv) {

  ros::init(argc, argv, "runner");
  ros::NodeHandle n;

  ros::Publisher publisher =
      n.advertise<geometry_msgs::PointStamped>("runner_position", 1000);

  ros::Rate loop_rate(10);

  geometry_msgs::PointStamped position_msg;

  position_msg.point.x = 1.0;
  position_msg.point.y = 1.0;
  position_msg.point.z = 1.0;

  while (ros::ok()) {

    position_msg.header.stamp = ros::Time::now();
    position_msg.point.x = position_msg.point.x + 1;
    position_msg.point.y = position_msg.point.y + 1;
    position_msg.point.z = position_msg.point.z + 1;

    publisher.publish(position_msg);
  }

  return 0;
}

File: src/task_1/runner.py
#!/usr/bin/env python
import rospy
from geometry_msgs.msg import Point

def runner():
    pub = rospy.Publisher('runner_position', Point, queue_size = 10)
    rospy.init_node('runner', anonymous=True)
    rate = rospy.Rate(10)

    position = Point()

    position.x = 1.0;
    position.y = 1.0;
    position.z = 1.0;

    while not rospy.is_shutdown():
        pub.publish(position)
        position.x = position.x + 1
        position.y = position.y + 1
        position.z = position.z + 1
        rate.sleep()


if __name__ == '__main__':
    try:
        runner()
    except rospy.ROSInterruptException:    
        pass
